spdx-utils/src/main/kotlin/SpdxLicenseIdMapping.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.spdx
+
+import com.here.ort.spdx.SpdxLicense.*
+
+/**
+ * A mapping from misspelled SPDX license IDs to valid SPDX IDs.
+ */
+object SpdxLicenseIdMapping {
```

Comment:
To be honest, I'm not so happy about the name. This just maps strings to something exactly as `SpdxLicenseStringMapping` does. How about "SpdxIdStringToLicenseMapping" for this object, and rename "SpdxLicenseStringMapping" to "SpdxFullStringToExpressionMapping"?

Reply:
Let's discuss the naming offline next week, these classes are not used much, so it is easy to rename them later.

Reply:
Ok.